source/vibe/web/rest.d

```diff
@@ -1,12 +1,228 @@
 /**
-	Automatic REST interface and client code generation facilities.
-
-	Copyright: © 2012-2013 RejectedSoftware e.K.
-	License: Subject to the terms of the MIT license, as written in the included LICENSE.txt file.
-	Authors: Sönke Ludwig, Михаил Страшун
-*/
+ * Module to work with REST API, from both server and client side.
+ *
+ * This modules aims to provide a typesafe way to deal with REST API.
+ * As they are quite a lot of similarities between them, this module use
+ * D interfaces as the way to represent REST API.
+ * While REST is not HTTP specific, the vast majority of use cases rely on it,
+ * and so does this module.
+ *
+ * This module, while being usable and more convenient from both client and
+ * server side, is also usable solely for its client or server implementation.
+ * The following documentation will explain in details how this module work,
+ * without any specifics about the client or server side.
+ * Since each implementation only require the use of one symbol from here,
+ * specifics are explained on those. You can thus find server only
+ * documentation on $(D registerRestInterface) and client only documentation
+ * on $(D RestInterfaceClient).
+ *
+ * There are few advantages in using this module over the more basic `vibe.http`
+ * package provided:
```

Comment:
I went for:

```diff
- * There are few advantages in using this module over the more basic `vibe.http`
- * package provided:
+ * There are few advantages in using this module over manual handling:
```

Not mentioning http at all feels like the best course of action after all.